app/views/casa_admin_mailer/deactivation.html.erb

```diff
@@ -1,17 +1,45 @@
-<meta itemprop="name" content="Confirm Email" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-<table width="100%" cellpadding="0" cellspacing="0" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-  <tr style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
-    <td class="content-block" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
+<style>
+  #table {
+    width: 100%;
+    border-spacing: 0;
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 120px;
+    margin: 0;
+  }
+
+    #table tbody tr {
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 14px;
+    margin: 0;
+  }
+  
+  .content-block {
+    font-family: Helvetica,Arial,sans-serif;
+    box-sizing: border-box;
+    font-size: 14px;
+    vertical-align: top;
+    margin: 0;
+    padding: 0 0 20px;
+  }
+
+</style>
+
+<meta itemprop="name" content="Confirm Email">
+<table id="table">
+  <tr id="tr">
```

Comment:
typically in css an id is only for one element on the page if you'd like to apply a style to multiple elements use a [class](https://www.w3schools.com/cssref/sel_class.asp)  
  
these ids are actually not being used because you're referring to the `<tr>`s with `#table tbody tr {` and not `#table tbody #tr {` so you can just delete them

Reply:
Thanks for the hints.